### PR TITLE
Blog post: Sprint 37 & 38 @Linagora

### PR DESCRIPTION
**Pull request by chibenwa**
A bit sad but that is the best formatting I found.

I don't want to invest more on this, if you agree.

capture d ecran de 2018-06-06 15-50-43

capture d ecran de 2018-06-06 15-50-48

```
To users, devs
Subject: Use the James blog to communicate about your uses of James

Hi everyone,

We took the decision, @Linagora, to be communicating more on our Sprint contents (finished and to upcoming). We consider the best place to do so is to do it via a blog post, for it to be more durable.

Hence, you can be reading about our past Sprint #37 and our upcoming Sprint #38 works : https://XXX

Furthermore, anybody using James is encouraged to do some posts on the same medium about your uses of James and work on it. To do so add a file in this folder [1] then open a Pull Request on this repository [2].

Best regards,

Benoit Tellier

[1] https://github.com/apache/james-project/tree/master/src/homepage/_posts
[2] https://github.com/apache/james-project
```

### CODE REVIEW

**Comment by rouazana** [comment]
s/finished and to upcoming/finished and upcoming/ ?
s/ :/:/

**Comment by chibenwa** [comment]
What do other team members thinks on this?

@aduprat  @mbaechler  ?

**Comment by aduprat** [comment]
> This post is very Linagora-oriented. Do we really need to expose the sprint number et the content of the particular sprint? Why not just listing the new features on a per-month basis for example?
:+1:

**Comment by chibenwa** [comment]
```
To users, devs
Subject: Use the James blog to communicate about your uses of James

Hi everyone,

We took the decision, @Linagora, to be communicating more on our work on the James project. We consider the best place to do so is to do it via a blog post, for it to be more durable.

Hence, you can be reading about May results and June plans: https://XXX

Furthermore, anybody using James is encouraged to do some posts on the same medium about your uses of James and work on it. To do so add a file in this folder [1] then open a Pull Request on this repository [2].

Best regards,

Benoit Tellier

[1] https://github.com/apache/james-project/tree/master/src/homepage/_posts
[2] https://github.com/apache/james-project
```

File: src/homepage/_posts/2018-06-06-linagora-june-nl.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+We need to work on some specific feature to make James easier to adopt for large organizations.
+
+# What was achieved in may
+
+## Notifications when soon over quota
+
+> A mail needs to be sent to the user to warn him. This is an attempts to allow clients not implementing quota's RFCs to still get the warning.
+>
+> This is implemented with mailbox listeners (and not mailets), with event sourcing. We added the interfaces and implementation for implementing features in James using event sourcing with events being stored in Cassandra or in memory.
```
Comment (rouazana) [inline src/homepage/_posts/2018-06-06-linagora-june-nl.markdown]: s/implementation/implementations
BTW implementation for implementing?

File: src/homepage/_posts/2018-06-06-linagora-june-nl.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+>
+> This mailbox listener is optional. We allowed one to register his (potentially custom) mailbox listeners when working with Guice.
+>
+> Most of the work was done during Sprint #36 but we finished this feature.
```
Comment (rouazana) [inline src/homepage/_posts/2018-06-06-linagora-june-nl.markdown]: still a sprint reference

File: src/homepage/_posts/2018-06-06-linagora-june-nl.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Linagora June newsletter"
+date:   2018-06-06 11:30:00 +0700
+categories: james update
+---
+
+In the name of the James team @Linagora I will be presenting you what we did in the past few weeks, and plan to work on in the coming weeks.
+
+We need to work on some specific feature to make James easier to adopt for large organizations.
```
Comment (rouazana) [inline src/homepage/_posts/2018-06-06-linagora-june-nl.markdown]: need?

File: src/homepage/_posts/2018-06-06-linagora-june-nl.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Data Loss Prevention
+
+> We should have a matcher applying rules defined by the admin. This matcher will then store suspicious emails in specific mail repository for an admin review.
+>
+> To implement this, we need to enhance browsing to MailRepository via WebAdmin. We need to allow storing such rules as well as updating them via webadmin. We need finally a mailet storing emails in a repository given its sender domain.
+
+### Exporting a mail account
+
+> As a user, I should be able to export the content of my mail account. This both allows easier migrations, comply with legal requirements as well as allow some forms of backups.
```
Comment (rouazana) [inline src/homepage/_posts/2018-06-06-linagora-june-nl.markdown]: I think it's only for admin as we'll implement it for now

File: src/homepage/_posts/2018-06-06-linagora-june-nl.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+>
+> The chosen format is emls in a ZIP format, with the folder structure, and specific metadata to ease restoring a James account (flags, etc...) . An admin (via webamin) should be able to download/upload such backups in order to do export/restores.
```
Comment (rouazana) [inline src/homepage/_posts/2018-06-06-linagora-june-nl.markdown]: EMLs?

Comment (rouazana) [reply src/homepage/_posts/2018-06-06-linagora-june-nl.markdown]: you can specify that folder structure will be in meta data, it's confusing in this sentence

Comment (rouazana) [reply src/homepage/_posts/2018-06-06-linagora-june-nl.markdown]: s/) ./)./

**Comment by chibenwa** [comment]
Merged

File: src/homepage/_posts/2018-06-06-linagora-june-nl.markdown [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+## Performance enhancements
+
+> We did experience slow requests with our Tika server. We contributed [this issue] which significantly enhanced Tika server performance, and we introduce a cache on James side to reduce calls to tika.
+>
+> We also worked on improving massive JMAP operation performance (delete a large number of emails).
+
+# What we will work on in June
+
+## Data Loss Prevention
```
[the remark this answers is not in the document]
Comment (rouazana) [reply src/homepage/_posts/2018-06-06-linagora-june-nl.markdown]: FYI both can apply, see https://en.wikipedia.org/wiki/Data_loss_prevention_software
But I agree that in this case Leak seems more appropriate.

**Pull request closed without merging**